lib/cdo/contact_rollups.rb

```diff
@@ -420,11 +420,23 @@ def self.update_professional_learning_attendance
             FROM #{DASHBOARD_DB_NAME}.workshop_attendance AS workshop_attendance
               INNER JOIN #{DASHBOARD_DB_NAME}.segments AS segments ON segments.id = workshop_attendance.segment_id
               INNER JOIN #{DASHBOARD_DB_NAME}.workshops AS workshops ON workshops.id = segments.workshop_id
-            ) q1
-          ORDER BY 1,2
-          ) q2
-        GROUP BY teacher_id
-      ) src
+            UNION
+            SELECT users.id,
+              CASE section_type
+                WHEN 'csins_workshop' THEN 'CS in Science'
+                WHEN 'csina_workshop' THEN  'CS in Algebra'
+                WHEN 'ecs_workshop' THEN 'Exploring Computer Science'
+                WHEN 'csp_workshop' THEN 'CS Principles'
+                WHEN 'csf_workshop' THEN 'CS Fundamentals'
+              END AS course
+            FROM #{DASHBOARD_DB_NAME}.sections
```

Comment:
Do you want to restrict to `sections` having one of the above `section_type`s? Otherwise, you'll be grabbing all teachers that are students, be it of a PD workshop or of a regular class.